.travis.yml

```diff
@@ -1,29 +1,35 @@
 language: go
 
 os:
-  - linux
-  - osx
+- linux
+- osx
 
 go:
-  - "1.12.x"
+- 1.13.x
 
 env:
+  global:
   - GO111MODULE=on
+  - GOPROXY=https://proxy.golang.org
```

Comment:
passing by :wink: 

isn't it default? why is it needed?

Reply:
Is it the default? I'm not sure.

Reply:
> The GOPROXY environment variable may now be set to a comma-separated list of proxy URLs or the special token direct, and its default value is now https://proxy.golang.org,direct

from release nodes of 1.13

Reply:
for convenience: https://golang.org/doc/go1.13#modules

Reply:
Ah.. I also found something here https://arslan.io/2019/08/02/why-you-should-use-a-go-module-proxy/

Guess I will remove it then :)
Thanks for spotting this!

Reply:
happy to be helpful how minor it is.